src/Components/Facility/HospitalList.tsx

```diff
@@ -378,20 +378,28 @@ export const HospitalList = (props: any) => {
                           ) && (
                             <div
                               key={i}
-                              className="bg-primary-100 text-primary-600 font-semibold px-2.5 py-0.5 rounded-md text-sm leading-5"
+                              className="flex gap-1 items-center bg-primary-100 text-primary-600 font-semibold px-2.5 py-0.5 rounded-md text-sm leading-5"
                               title={
                                 FACILITY_FEATURE_TYPES.filter(
                                   (f) => f.id === feature
                                 )[0]?.name
                               }
                             >
-                              <i
-                                className={`fas fa-${
-                                  FACILITY_FEATURE_TYPES.filter(
-                                    (f) => f.id === feature
-                                  )[0]?.icon
-                                }`}
-                              />{" "}
+                              {typeof FACILITY_FEATURE_TYPES.filter(
+                                (f) => f.id === feature
+                              )[0]?.icon === "string" ? (
+                                <i
+                                  className={`${
+                                    FACILITY_FEATURE_TYPES.filter(
+                                      (f) => f.id === feature
+                                    )[0]?.icon
+                                  }`}
+                                />
+                              ) : (
+                                FACILITY_FEATURE_TYPES.filter(
+                                  (f) => f.id === feature
+                                )[0]?.icon
+                              )}{" "}
```

Comment:
we are filtering it 2 times and repeating the code 3 times, can we export this into a function something like,

```js
      const getFacilityFeatureIcon = (featureId: number) => {
        const feature = FACILITY_FEATURE_TYPES.find(
          (f) => f.id === featureId
        );
  
        if (!icon) return null;
  
        return typeof feature.icon === "string" ? (
          <i
            className={feature.icon}
          />
        ) : (
          feature.icon
        )}{" "}
        &nbsp;
        {
          feature.name
        }
      }                       
```